mypyc/emitter.py

```diff
@@ -642,6 +671,15 @@ def visit_load_int(self, op: LoadInt) -> None:
         dest = self.reg(op.dest)
         self.emit_line('%s = %d;' % (dest, op.value * 2))
 
+    def visit_module_get(self, op: ModuleGet) -> None:
+        dest = self.reg(op.dest)
+        self.emit_line('%s = %s;' % (dest, c_module_name(op.module)))
+
+    def visit_get_attr(self, op: GetAttr) -> None:
+        dest = self.reg(op.dest)
+        left = self.reg(op.left)
+        self.emit_line('{} = CPyObject_GetAttrString({}, "{}");'.format(dest, left, op.right))
```

Comment:
This would likely be faster if we'd use a string object for the attribute name. At least add TODO comment to look into this.